test/__tests__/user_transformation.test.js

```diff
@@ -243,6 +243,34 @@ describe("User transformation", () => {
     expect(output[0].transformedEvent.dummy_key).toEqual(secrets.dummy_key);
   });
 
+  it(`Simple ${name} Test with credentials for codeVersion 1`, async () => {
```

Comment:
Let's add test for what happens when we have no lookup found, missing credentials in ivm etc right ?